frontend/src/features/MainWindow/slice.ts

```diff
@@ -1,24 +1,30 @@
 import { createSlice } from '@reduxjs/toolkit'
 
+import { UserType } from '../../domain/entities/beaconMalfunction/constants'
+import { getOnlyVesselIdentityProperties, vesselsAreEquals } from '../../domain/entities/vessel/vessel'
 import { getLocalStorageState } from '../../utils'
-import { UserType } from '../entities/beaconMalfunction/constants'
-import { getOnlyVesselIdentityProperties, vesselsAreEquals } from '../entities/vessel/vessel'
 
-import type { MapBox } from '../entities/map/constants'
+import type { MapBox } from '../../domain/entities/map/constants'
 import type { PayloadAction } from '@reduxjs/toolkit'
+import type { VesselIdentity } from 'domain/entities/vessel/types'
 
 const userTypeLocalStorageKey = 'userType'
 const lastSearchedVesselsLocalStorageKey = 'lastSearchedVessels'
 
-// TODO Properly type this redux state.
-export type GlobalState = {
+export type MainWindowState = {
   blockVesselsUpdate: boolean
   error: any
   // TODO Rename this prop.
   healthcheckTextWarning: string[]
   isBackoffice: boolean
   isUpdatingVessels: boolean
   lastSearchedVessels: any[]
+  leftDialog:
```

Comment:
```suggestion
  openedLeftMapDialog:
```
On a déjà une variable `leftMapBoxOpened` (l. 28), à enrichir ?

Reply:
Je trouve que le préfixe `opened` clarifie en effet ^^. Par contre le `Map` je ne sais pas trop car on est pluyôt sur la MainWindow pour moi, dans laquelle il se trouve que la Map prend tout l'espace (ce qui est sensé être "clair" quand on écrit `mainWindow.openedLeftDialog`), non ?